### ISSUE TITLE
Moonlight Hijacks audio if closed via ctl-shift-Q

### ISSUE THREAD

**Issue by theFisher86** [moonlight-stream/moonlight-chrome#196]
This is across all moonlight clients, not just moonlight-chrome.  If you abruptly quit out of moonlight it hijacks the audio on the host pc.  If you test audio via the little "bell" sound that is made when you change the volume it still works but games, kodi video and webpages no longer play audio.  Restarting the computer is the only way to restore it.  However if you stream again the client has audio.  Seems to be caused whenever moonlight isn't able to shutdown properly.  While a proper fix may not be possible I would be interested in knowing what application needs to be "restarted" on the pc or force-closed to restore audio without a full system restart?

**Comment by jacobmix** [comment]
I've had the same problem here: https://github.com/moonlight-stream/moonlight-chrome/issues/193
And i found out how you can get your audio back:
Start Task Manager>Show processes from all users (Admin)>nvstream.exe>End Process.
I'm on Windows 7. But should be about the same if you're on 10 or something.

**Comment by raidancampbell** [comment]
I wouldn't call this a Moonlight bug at all: when streaming, GameStream is rerouting the audio to a virtual device, which is sent to the client.  If you abruptly quit moonlight or simply `ctrl`+`alt`+`shift`+`q` out of it, the stream is still active.  GameStream is still routing the audio to this virtual device.

Testing on Win10, properly exiting the current running app by pressing `Quit Current App` rerouted the audio to the previous device.  Alternatively, I was able to simply select the physical audio output device from the server itself.  Another option is to stream with `Remote Audio` set to on, which will play audio on both the server and the client.

tl;dr
 - `Quit Current App` from the client will solve this
 - reselecting the proper output device on the server will solve this
 - streaming with remote audio on will allow you to leave the stream open and still play audio on the server.

And if all else fails, killing `nvstreamer.exe` is a fine choice.
